Tasks/ContainerStructureTestV1/task.loc.json

```diff
@@ -0,0 +1,60 @@
+{
+  "id": "39bc2c9b-55b7-4835-89cd-6cc699ef7220",
+  "name": "ContainerStructureTest",
+  "friendlyName": "ms-resource:loc.friendlyName",
+  "description": "ms-resource:loc.description",
+  "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/test/publish-test-results",
+  "helpMarkDown": "ms-resource:loc.helpMarkDown",
+  "category": "Test",
+  "visibility": [
+    "Build",
+    "Release"
+  ],
+  "author": "Microsoft Corporation",
+  "version": {
+    "Major": 1,
```

Comment:
Make this 0. The task should be in preview mode

Reply:
Done